### DISCUSSION TITLE
Can you use the bot for modded servers?

### DISCUSSION THREAD

**Discussion by SoakedSun** [MrKinau/FishingBot#213] [Q&A]
My friend is hosting a modded server w a few light mods, just to mostly add a little bit of QoL changes to the vanilla experience. Is there a way to use the bot on that server?

**Comment by MrKinau** [comment]
Yes you might be able to use the bot on Fabric modded servers just like on vanilla servers (unless a mod requires some custom handshake). Forge won't be supported.
